### ISSUE TITLE
[JENKINS-33947] setup wizzard "Dependencies" is not navigable with keyboard

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/jenkins#16973]
<p>start the new wizard and choose customize plugins.</p>

<p>try and expand the dependencies of a plugin without using a mouse/touchscreen</p>

<h3><a name="expectedresults"></a>expected results</h3>

<p>it works</p>

<h3><a name="actualresults"></a>actual results</h3>

<p>epic failure.<br/>
Accessibility issue.</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/teilo.png" title="teilo's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=teilo">teilo</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-33947" target="_blank">setup wizzard "Dependencies" is not navigable with keyboard</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/kzantow.png" title="kzantow's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=kzantow">kzantow</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Major
<li><b>component(s)</b>: core
<li><b>label(s)</b>: 2.0, 2.0-beta, accessibility, community-bee, ui, ux
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2016-05-27T13:29:33+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 3
<li><b>imported</b>: 2025-11-24
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>start the new wizard and choose customize plugins.</p>

<p>try and expand the dependencies of a plugin without using a mouse/touchscreen</p>

<h3><a name="expectedresults"></a>expected results</h3>

<p>it works</p>

<h3><a name="actualresults"></a>actual results</h3>

<p>epic failure.
Accessibility issue.</p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>windows chrome 2.0-beta-1</code></li></ul>

**Comment by jenkins-infra-bot** [comment]
- _Remote link associated with this issue:_


  - [PR 2294](https://github.com/jenkinsci/jenkins/pull/2294)

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/danielbeck.png" title="danielbeck's avatar" /> danielbeck:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-33947?focusedId=252607&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-252607">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Tab navigation/element focus is also kind of weird otherwise.</p>
</pre>
</details>

</ul>
</details>

<p>Tab navigation/element focus is also kind of weird otherwise.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/scm_issue_link.png" title="scm_issue_link's avatar" /> scm_issue_link:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-33947?focusedId=259117&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-259117">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Code changed in jenkins
User: Keith Zantow
Path:
 war/src/main/js/pluginSetupWizardGui.js
 war/src/main/js/templates/pluginSelectList.hbs
 war/src/main/less/pluginSetupWizard.less
<a href="http://jenkins-ci.org/commit/jenkins/50c704957dbb79cc8c2a6e454628052ef46a5039" class="external-link" target="_blank" rel="nofollow noopener">http://jenkins-ci.org/commit/jenkins/50c704957dbb79cc8c2a6e454628052ef46a5039</a>
Log:
  <span class="error">&#91;FIX JENKINS-33947&#93;</span> - Fix keyboard navigation in setup wizard (#2294)</p>


</pre>
</details>

</ul>
</details>

<p>Code changed in jenkins<br/>
User: Keith Zantow<br/>
Path:<br/>
 war/src/main/js/pluginSetupWizardGui.js<br/>
 war/src/main/js/templates/pluginSelectList.hbs<br/>
 war/src/main/less/pluginSetupWizard.less<br/>
<a href="http://jenkins-ci.org/commit/jenkins/50c704957dbb79cc8c2a6e454628052ef46a5039" class="external-link" target="_blank" rel="nofollow noopener">http://jenkins-ci.org/commit/jenkins/50c704957dbb79cc8c2a6e454628052ef46a5039</a><br/>
Log:<br/>
  <span class="error">&#91;FIX JENKINS-33947&#93;</span> - Fix keyboard navigation in setup wizard (#2294)</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/oleg_nenashev.png" title="oleg_nenashev's avatar" /> oleg_nenashev:</i></summary>

<ul>
<li><i>Original <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-33947?focusedId=260109&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-260109">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Released as Jenkins-2.7</p>
</pre>
</details>

</ul>
</details>

<p>Released as Jenkins-2.7</p>
